### PR TITLE
Fixed managed variants search by description

### LINKED ISSUE #3160: Filter managed variants for description?
A box exists but..

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3160 
- Introduced ignore case in the search by description

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. Load 2 variants into managed variants, **one** with a description.
2. Filter by one word present in the variant description
3. The search should return only one variant

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by DN

### CODE REVIEW

File: scout/server/blueprints/managed_variants/controllers.py [hunk 1]
```diff
@@ -66,6 +66,10 @@ def managed_variants(request):
         subcat[0] for subcat in SUBCATEGORY_CHOICES
     ]:
         query_options["sub_category"].append(sub_cat)
+
+    if request.form.get("description") is not None and request.form.get("description") != "":
```
Comment (northwestwitch) [inline scout/server/blueprints/managed_variants/controllers.py:70]: This is the actual fix

Comment (dnil) [reply scout/server/blueprints/managed_variants/controllers.py:70]: Hihi, it's a little bit like when one finds the cable not being plugged in! Well done!

**Review by dnil — APPROVED** [review]
💯

**Comment by dnil** [comment]
Before:
⛔ Screenshot 2022-02-15 at 09 21 15

After:
🆗 Screenshot 2022-02-15 at 09 20 31

**Pull request merged**